src/test/resources/db/scripts/ProjectRepositoryIT_before.sql

```diff
@@ -2,10 +2,10 @@ INSERT INTO jhi_user(login, email, password_hash, activated, created_by)
 VALUES ('ProjectRepositoryIT-user-1', 'ProjectRepositoryIT-user-1@localhost', 'some-password-hash-whatever', true, 'me'),
        ('ProjectRepositoryIT-user-2', 'ProjectRepositoryIT-user-2@localhost', 'some-password-hash-whatever', true, 'me');
 
-INSERT INTO project(name)
-VALUES ('ProjectRepositoryIT-project-1'),
-       ('ProjectRepositoryIT-project-2'),
-       ('ProjectRepositoryIT-project-3');
+INSERT INTO project(name, start_time, end_time)
+VALUES ('ProjectRepositoryIT-project-1', now(), now()),
+       ('ProjectRepositoryIT-project-2', now(), now()),
+       ('ProjectRepositoryIT-project-3', now(), now());
 
 INSERT INTO INVITATION(email, accepted, project_id, role_name, jhi_user_id)
 VALUES ('ProjectRepositoryIT-user-1@localhost', true, (select id from project where name = 'ProjectRepositoryIT-project-1'), 'VIEWER',
```

Comment:
Würde den Testusers auch noch je ein Projekt geben mit der Rolle ADMIN. Somit könnten wir auch die Adminfeatures überprüfen

Reply:
hier geht es um die Testdaten für einen einzigen Test (`ProjectRepositoryIT`). `Role:ADMIN` ist in diesem Fall nicht von Belang. der Test möchte nur die Liste von `Project`s die über die `Invitation` verlinkt sind abfragen, respektive diese Funktionalität testen.